pyiron/sphinx/base.py

```diff
@@ -1242,6 +1243,38 @@ def write_input(self):
             f.write("include <parameters.sx>;\n\n")
             f.write(self.input.sphinx.to_sphinx(indent=0))
 
+    def get_charge_density(self):
```

Comment:
If the way we store this is compatible with VASP, we could even shift this to `GenericDFTJob` and raise a `NotImplementedError` for other codes.